src/python/ETXinitPassthrough.py

```diff
@@ -2,24 +2,21 @@
 import subprocess
 import argparse
 import serials_find
-
-def dbg_print(line=''):
-    sys.stdout.write(line + '\n')
-    sys.stdout.flush()
-
 try:
     import streamexpect
 except ImportError:
-    sys.stdout.write("Installing pexpect")
+    sys.stdout.write("Installing streamexpect")
     subprocess.check_call([sys.executable, "-m", "pip", "install", "streamexpect"])
     try:
         import streamexpect
     except ImportError:
-        env.Execute("$PYTHONEXE -m pip install streamexpect")
-        try:
-            import streamexpect
-        except ImportError:
-            streamexpect = None
+        raise SystemExit("Please, install 'streamexpect' and try again.")
```

Comment:
Do you think we should just copy streamexpect into our codebase? It is a single file after all. I really don't think we can expect users to understand how to install it.

Reply:
Exactly, thats why I think maybe we just copy the one file (which hasn't changed since 2016) into the code.

Reply:
That's fine by me too.

Reply:
I think it makes sense to copy those files.
I had to remove that other install call since env cannot be imported...

Reply:
I "installed" both `streamexpect`and `six` libraries locally.